tests/PHPStan/Analyser/data/bug-6866.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Bug6866;
+
+use function PHPStan\Testing\assertType;
+
+$str = str_repeat('a', 99);
```

Comment:
Code in data/* files should be within functions or class-methods, so it won't be executed while running tests, but just been analyzed phpstan.

Otherwise looks great

Reply:
ah ah this is real-time feedback :)

thank you for your help!